src/infra/azure/cosmos/config.ts

```diff
@@ -0,0 +1,27 @@
+import * as t from "io-ts";
+
+import * as RE from "fp-ts/ReaderEither";
+import { pipe } from "fp-ts/function";
+
+import { sequenceS } from "fp-ts/lib/Apply";
+
+import { readFromNodeEnv } from "../../../app/env";
+
+export const CosmosConfig = t.type({
+  connectionString: t.string,
+  dbName: t.string,
+});
+
+export type CosmosConfig = t.TypeOf<typeof CosmosConfig>;
+
+export const getCosmosConfigFromEnvironment: RE.ReaderEither<
+  NodeJS.ProcessEnv,
+  Error,
+  CosmosConfig
+> = sequenceS(RE.Apply)({
+  connectionString: readFromNodeEnv("CosmosDbConnectionString"),
+  dbName: pipe(
+    readFromNodeEnv("CosmosDbDatabaseName"),
```

Comment:
ditto

Reply:
These are the names of the environment variables; they aren't utilized anywhere else, so declaring them as constants would be redundant.